### DISCUSSION TITLE
Option to hide Achievements and Sponsorships on the profile page

### DISCUSSION THREAD

**Discussion by jstolarek** [community/community#6270] [Other Feature Feedback, Questions, & Ideas] [answered]
I just sponsored a project and now my profile page is cluttered with badges: one for sponsorship, the other for achievement. I need an option to hide these badges, especially Achievements, which are total nonsense. Worst of all, these two badges display above the list my Organizations, moving information that is important and accessed frequently to a less accessible place. In the end GitHub is a tool for doing my work and these badges make it less useful and more cluttered - Achievements in particular. I'm sure I can remove the sponsorship badge by cancelling the sponsorship, but will this remove the Achievements as well?

EDIT: I realized I can hide the sponsorship information, but that still shows empty Sponsorship section on my profile and doesn't hide the Achievements.

**Comment by ashleygwilliams** [comment]
I'd like to second this request. I am not personally interested in my GitHub interactions being gamified and I find it noisy and stressful.

**Reply by ashleygwilliams** [reply]
This was not meant as an answer, but an effort to encourage attention to this...

**Comment by bailey** [comment, marked as answer]
Hi @jstolarek - Achievements are an optional feature that can be turned off from your `Profile settings` in https://github.com/settings. Let me know if that helps!

**Reply by jstolarek** [reply]
Indeed, this seems to have been implemented since I originally opened this discussion.

**Comment by Jackenmen** [comment]
I kind of like the achievements but same as the author, I feel that ideally, the organization list would show above them.

**Comment by MrKrishnaAgarwal** [comment]
![image](https://user-images.githubusercontent.com/100597998/173042312-a45c71be-bdfe-4be9-8309-8033a7cae677.png)
You can turn off Show Achievements on your profile by going into settings

**Comment by jfullstackdev** [comment]
latest update, you can now pick what you want to hide, unlike in the past, it was all or nothing

go to Achievements Tab, click the `Achievements` word, it is a link

![image](https://user-images.githubusercontent.com/47092464/204711684-e7c5ebb6-57f9-4fbf-b8fe-96c2182f41bb.png)

or this URL, it's same thing 

https://github.com/[username]?tab=achievements

you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711801-dde96c4d-5133-4201-ad8a-9b23b55ed5d2.png)

then, click any badge, hover left then you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711940-2c5e8b4d-7580-4145-afc0-268f1739d8de.png)

then boom ... you now have the option to hide those badges that you want to hide

**Comment by exanrauzer** [comment]
@jdevstatic How to navigate to achievements tab?
